.travis.yml

```diff
@@ -47,7 +54,7 @@ before_install:
   - rsync -az ${TRAVIS_BUILD_DIR}/ $HOME/gopath/src/github.com/alphagov/paas-cf/
   - export TRAVIS_BUILD_DIR=$HOME/gopath/src/github.com/alphagov/paas-cf
   - cd $HOME/gopath/src/github.com/alphagov/paas-cf
-  - cd scripts && BUNDLE_GEMFILE=Gemfile bundle install --jobs=3 --retry=3 --deployment && cd ..
+  - bundle install --jobs=3 --retry=3 --deployment
```

Comment:
This is unnecessary because Travis already takes care of installing the top-level Gemfile.